app/containers/Applications/ApplicationDetailsPdf.tsx

```diff
@@ -0,0 +1,314 @@
+    fontSize: 12,
+    position: 'absolute',
+    bottom: 20,
+    left: 0,
+    right: 0,
+    textAlign: 'center'
+  }
+});
+
+const arrayFieldTemplate = props => {
+  return (
+    <>
+      <Text>
+        {props.title === 'gases' || props.title === 'Source Types'
+          ? null
+          : props.title}
+      </Text>
+      {props.items.map(item => {
+        return <Text key={item.index}>{item.children}</Text>;
+      })}
+    </>
+  );
+};
+
+const objectFieldTemplate = props => {
+  return (
+    <>
+      {props.properties.map(
+        prop =>
+          prop.content && <View key={prop.content.key}>{prop.content}</View>
+      )}
+      {!props.properties && '\n'}
+    </>
+  );
+};
+
+const fieldTemplate = props => {
+  return (
+    <>
+      {props.label && (
+        <View style={styles.label}>
+          {props.label === 'gases' ||
+          props.label === 'sourceTypeName' ? null : props.classNames.includes(
+              'field-object'
+            ) ? (
+            <Text style={{fontSize: 15, letterSpacing: 2}}>
+              {`\n\n${props.label}: `}
+            </Text>
+          ) : (
+            <Text>{`\n${props.label}: `}</Text>
+          )}
+        </View>
+      )}
+      {props.children && <Text style={styles.fields}>{props.children}</Text>}
+    </>
+  );
+};
+
+const emissionGasFieldTemplate = props => {
+  return (
+    <Text>
+      {'\n'}
+      {props.gasType} - Tonnes: {props.annualEmission} - Tonnes (CO2e):{' '}
+      {props.annualCO2e}
+    </Text>
+  );
+};
+
+const productionFieldTemplate = props => {
+  return (
+    <>
+      <Text style={{fontSize: 15, letterSpacing: 2}}>
+        {'\n'}Production:{'\n'}
+      </Text>
+      <Text>
+        Product allocation factor (%): {props.productionAllocationFactor}
+        {'\n'}
+      </Text>
+      <Text>
+        Annual Production Amount: {props.productUnits}
+        {'\n'}
+      </Text>
+      <Text>
+        Units: {props.quantity}
+        {'\n'}
+      </Text>
+      <Text>{props.productRowId}</Text>
+    </>
+  );
+};
+
+const CUSTOM_FIELDS = {
+  StringField: props => <Text>{props.formData ?? '[Not Entered]'}</Text>,
+  emissionSource: props => (
+    <Text style={{fontSize: 13}}>
+      {'\n\n'}
+      {props.formData}
+    </Text>
+  ),
+  emissionGas: props => emissionGasFieldTemplate(props.formData),
+  production: props => productionFieldTemplate(props.formData)
+};
+
+const customWidget = {
+  RadioWidget: props =>
+    props.value === true ? <Text>Yes</Text> : <Text>No</Text>
+};
+
+export const ApplicationDetailsPdf: React.FunctionComponent<Props> = props => {
+  const {application} = props;
+  const facility = application.facilityByFacilityId;
+  const formResults = application.formResultsByApplicationId.edges;
+
+  const pdfFilename = `CIIP_Application_${application.facilityByFacilityId.facilityName}_${application.reportingYear}.pdf`;
+
+  const pdfDocument = (
+    <Document>
+      <Page wrap size="A4" style={styles.page}>
```

Comment:
pretty sure we're using the `letter` format in north america

Reply:
Updated